config.default.yaml

```diff
@@ -251,6 +251,7 @@ sector:
   electricity_distribution_grid: true
   electricity_distribution_grid_cost_factor: 1.0  #multiplies cost in data/costs.csv
   electricity_grid_connection: true  # only applies to onshore wind and utility PV
+  electricity_grid_transmission_losses: 3 # Enable (=3) or disable (= 0) transmission losses in electricity grid
```

Comment:
2 tangents is probably sufficient (or max we can computationally afford)

Reply:
Let's go with disabled by default and I'll run a few comparisons